prdoc/pr_3199.prdoc

```diff
@@ -0,0 +1,12 @@
+# Schema: Polkadot SDK PRDoc Schema (prdoc) v1.0.0
+# See doc at https://raw.githubusercontent.com/paritytech/polkadot-sdk/master/prdoc/schema_user.json
+
+title: sp-std to core
+
+doc:
+  - audience: Node Dev
```

Comment:
Runtime devs also care about this afaik?

Reply:
This doesn't require any prdoc at all. This is a purely internal change that doesn't change anything to the outside.

Reply:
Yes, good point. Only when `sp_std` itself is removed or marked deprecated we would need a prdoc and possibly some doc updates.

Reply:
Exactly.

Reply:
Have removed prdoc.